src/main/scala/com/databricks/spark/avro/DefaultSource.scala

```diff
@@ -72,7 +72,18 @@ private[avro] class DefaultSource extends FileFormat with DataSourceRegister {
       }
     }
 
-    val avroSchema = {
+    // User can specify an optional avro json schema.
+    // For example:
+    // {{{
+    //   val schema = new Schema.Parser().parse(new File("user.avsc"))
+    //   val spark = SparkSession.builder().master("local").getOrCreate()
+    //   spark
+    //     .read
+    //     .format("com.databricks.spark.avro")
+    //     .option(DefaultSource.AVRO_SCHEMA, schema.toString)
+    //     .load("/tmp/avro_file_path").show()
+    // }}}
```

Comment:
Nothing important, but you don't need to use `{{{ ... }}}` here since it's an inline comment.